docs/source/index.rst

```diff
@@ -12,4 +12,5 @@ Documentation of `sentinelhub` Python package
     logging
     examples
     tutorials
+    cdse
```

Comment:
should we group these three together or is it fine the way it is now?

Reply:
I was thinking of adding a .rst files named `Examples` which then includes these three subcategories (so that the sidebar is nicely grouped) unless you want the CDSE examples to be extra visible